i18n/en.json

```diff
@@ -5802,6 +5794,10 @@
     "id": "ent.elasticsearch.index_channel.error",
     "translation": "Failed to index the channel"
   },
+  {
+    "id": "ent.elasticsearch.index_channels_batch.error",
+    "translation": ""
+  },
```

Comment:
Was this actually missing or is it an out of sync issue?

Reply:
Out of sync issue. Fixed now.